fhttp/httprunner.go

```diff
@@ -59,20 +59,16 @@ type HTTPRunnerResults struct {
 // To be set as the Function in RunnerOptions.
 func (httpstate *HTTPRunnerResults) Run(ctx context.Context, t periodic.ThreadID) (bool, string) {
 	log.Debugf("Calling in %d", t)
-	code, body, headerSize := httpstate.client.Fetch(ctx)
-	size := len(body)
+	code, size, headerSize := httpstate.client.StreamFetch(ctx)
 	log.Debugf("Got in %3d hsz %d sz %d - will abort on %d", code, headerSize, size, httpstate.AbortOn)
 	httpstate.RetCodes[code]++
 	httpstate.sizes.Record(float64(size))
 	httpstate.headerSizes.Record(float64(headerSize))
 	if httpstate.AbortOn == code {
 		httpstate.aborter.Abort(false)
-		log.Infof("Aborted run because of code %d - data %s", code, DebugSummary(body, 1024))
+		log.Infof("Aborted run because of code %d - data len %d", code, size)
 	}
-	if code == http.StatusOK {
-		return true, "200"
-	}
-	return false, fmt.Sprint(code)
+	return codeIsOK(code), strconv.Itoa(code)
```

Comment:
that initial implementation when I added support for success/error of each run/access log was (my) bad

fixed now, thx for noticing @jneate